code/game/objects/structures/false_walls.dm

```diff
@@ -117,32 +118,28 @@
 	else if(istype(W, /obj/item/weapon/weldingtool))
 		var/obj/item/weapon/weldingtool/WT = W
 		if(WT.remove_fuel(0,user))
-			dismantle(user)
+			dismantle(user, TRUE)
 	else if(istype(W, /obj/item/weapon/gun/energy/plasmacutter))
-		dismantle(user)
+		dismantle(user, TRUE)
 	else if(istype(W, /obj/item/weapon/pickaxe/drill/jackhammer))
 		var/obj/item/weapon/pickaxe/drill/jackhammer/D = W
 		D.playDigSound()
-		dismantle(user)
+		dismantle(user, TRUE)
 	else
 		return ..()
 
-/obj/structure/falsewall/proc/dismantle(mob/user)
+/obj/structure/falsewall/proc/dismantle(mob/user, girder)
```

Comment:
I'd prefer a better arg name, "girder" makes me think the arg is supposed to be a girder object. "drop_girder" is more explicit.